### ISSUE TITLE
Image view, swiping doesn't work on the mobile device

### ISSUE THREAD

**Issue by mfilser** [wekan/wekan#5074]
### Server Setup Information
* Did you test in newest Wekan?: Yes (7.05)
* Deployment Method (Snap/Docker/Sandstorm/bundle/source): Docker
* Http frontend if any (Caddy, Nginx, Apache, see config examples from Wekan GitHub wiki first): Traefik
* MongoDB Version: 6
* What webbrowser version are you using (Wekan should work on all modern browsers that support Javascript)? Chrome (mobile)

### Problem description
Since #4995 swiping the picture left/right doesn't go to the next / previous picture, and swipe up doesn't close the picture on my mobile device. I am using a Samsung S20.

**Comment by xet7** [comment]
@mfilser 

Yes, we don't have swipebox anymore. Instead:
- we don't have those many errors at browser inspect console
- we have preview of PDF
- we have newest dependencies that do not have known vulnerabilities

**Comment by mfilser** [comment]
@xet7 
I know.

But maybe it could be implemented again?

@VidVidex 
Do you have time to look at it?

**Comment by xet7** [comment]
Yes, maybe @VidVidex could add some code to swipe attachments, without adding back that swipebox.

**Comment by xet7** [comment]
@mfilser 

> But maybe it could be implemented again?

Yes, after all these big upgrades, we are adding back features.

**Comment by VidVidex** [comment]
I'll try as soon as I have some time. I think this shouldn't be too difficult to implement.

**Comment by mfilser** [comment]
Great, thank you very much 🙂

**Comment by mfilser** [comment]
@VidVidex 
Thank you, it's working great :slightly_smiling_face:
